MicrowaveGame/ProjectSettings/ProjectSettings.asset

```diff
@@ -630,7 +630,7 @@ PlayerSettings:
     m_VersionCode: 1
     m_VersionName: 
   apiCompatibilityLevel: 6
-  activeInputHandler: 1
+  activeInputHandler: 2
```

Comment:
Looks like it switched the project back to mixing the old and new input systems. This should be changed to new input system only.